renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -22,9 +22,13 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#include <algorithm>
+#include <iterator>
 #include "vk_core.h"
 #include "vk_replay.h"
 #include "vk_resources.h"
+#include "driver/ihv/amd/amd_counters.h"
```

Comment:
You might fall afoul of clang-format telling you to move these headers above `vk_core.h` - if you need to maintain this order just add a space so they're two separate groups of includes.

Reply:
Fixed